deployment/apache.template.conf

```diff
@@ -1,11 +1,11 @@
 <VirtualHost *:80>
     ServerName evap
-    
+
     DocumentRoot /var/www/
-    
-    WSGIDaemonProcess wsgi processes=2 threads=15 display-name=%{GROUP} user=vagrant
-    WSGIProcessGroup wsgi
+
     WSGIScriptAlias / /vagrant/evap/wsgi.py
+    WSGIDaemonProcess evap processes=2 threads=15 display-name=%{GROUP} user=vagrant python-home=/vagrant/env
```

Comment:
Me no understand. We are not running vagrant in production, right? Why is this prepared for a venv in /vagrant/?

Reply:
Or is this for the development vagrant machine and should be changed to /home/vagrant/...?

Reply:
i forgot to change that and didn't test the apache server in the vm.